db/schema.rb

```diff
@@ -465,6 +465,12 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "picture"
+    t.boolean  "payed",                default: false
```

Comment:
Right spelling is paid

Reply:
If you have already run a migration, don't just go and change something in there. Because changes are not introduced in the schema file. 
You need to reverse the migration, make the changes, re-run it.
Of course that means that all of your values in those fields will be deleted. 
And this is obviously an option only for unmerged migrations.